### PR TITLE
update python base images

### PR DESCRIPTION
**Pull request by sallyom**
version of cmake in python39 is 3.20, but llama-cpp-python requires >3.21 
updating the base python image everywhere to keep consistency 

/cc @MichaelClifford 
/cc @lmilbaum 
/cc @jeffmaury

### CODE REVIEW

File: audio-to-text/client/Containerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM registry.access.redhat.com/ubi9/python-39:latest
+FROM registry.access.redhat.com/ubi9/python-311:latest
```
Comment (lmilbaum) [inline audio-to-text/client/Containerfile]: Is it possible to pin the version? Using the latest tag might cause some issues

Comment (lmilbaum) [reply audio-to-text/client/Containerfile]: Same comment is applied to everywhere the latest tag is being used. Renovate can be onboarded and used for bumping the versions.

**Comment by MichaelClifford** [comment]
Would it make sense to pin the version of `llama_cpp_python` that worked earlier instead changing the base image? Seems like updating the base image might have wide reaching affect throughout the project. 

Did we confirm all of the model servers and AI apps work with python 3.11?

**Comment by rhatdan** [comment]
Needs a rebase.

**Comment by sallyom** [comment]
I'm going to pin both llamacpp-python _and_ the python image to match what Liora has in her tests, sg?

**Comment by rhatdan** [comment]
Fine with me.

File: audio-to-text/client/Containerfile (hunk 1 above)
Comment (sallyom) [reply audio-to-text/client/Containerfile]: I updated to pin all python versions

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**